examples/h2o-classification/h2o-example.ipynb

```diff
@@ -0,0 +1,218 @@
+    "import os\n",
+    "sys.path.insert(0, os.path.join(os.getcwd(), \"..\", \"..\"))\n",
+    "import bentoml\n",
+    "\n",
+    "print(bentoml.__file__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# H2O Classification"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !curl -i https://raw.githubusercontent.com/multicode/h2o-notebook/master/prostate.csv -o prostate.csv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import h2o"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "h2o.init()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "prostate = h2o.import_file(path=\"https://raw.githubusercontent.com/multicode/h2o-notebook/master/prostate.csv\")\n",
+    "prostate.describe()\n",
+    "type(prostate)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# import the deep learning estimator module\n",
+    "from h2o.estimators.deeplearning import H2ODeepLearningEstimator\n",
+    "# transform the target variable into a factor\n",
+    "prostate[\"CAPSULE\"] = prostate[\"CAPSULE\"].asfactor()\n",
+    "# construct and define the estimator object \n",
+    "model = H2ODeepLearningEstimator(activation = \"Tanh\", hidden = [10, 10, 10], epochs = 100)\n",
+    "# train the model on the whole prostate dataset\n",
+    "model.train(x = list(set(prostate.columns) - set([\"ID\",\"CAPSULE\"])), y =\"CAPSULE\", training_frame = prostate)\n",
+    "model.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "predictions=model.predict(prostate)\n",
+    "predictions.show()\n",
+    "print(type(model))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "path = h2o.save_model(model=model, path=\"./model\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Export with BentoML"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile h2o_model_service.py\n",
+    "import pandas as pd\n",
+    "import h2o\n",
+    "import bentoml\n",
+    "from bentoml.artifact import H2oModelArtifact\n",
+    "from bentoml.handlers import DataframeHandler\n",
+    "\n",
+    "@bentoml.artifacts([H2oModelArtifact('model')])\n",
+    "@bentoml.env(conda_dependencies=[\"h2o\", \"pandas\"])\n",
```

Comment:
I think this env config actually won't install h2o properly, you will need to add the conda-forge channel and also `conda install conda-py` just glancing through the document.  would be great to include a docker build in this notebook as well, cause that way we can actually test these things

Reply:
yup just tested this out, 'conda install h2o' only installs the java package. It seems to also install the python package in python2.7 - an easy workaround is to make it a pip dependency, because `pip install h2o` seems to work.

Reply:
Yep. need to commit that change to reflect this.

Reply:
done